src/components/Header/Nav.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import './Nav.css';
+
+export default class HeaderNav extends Component {
+	render() {
+		return (
+			<nav>
+				<ul>
+					<li>Main</li>
```

Comment:
Are we using tabs spaces for tabulation? we were using two spaces in angular

Reply:
fixed